.env

```
@@ -1,6 +1,8 @@
-MONGODB_URI = "mongodb+srv://tsExpress:Karan204@cluster0.ticawrm.mongodb.net/BookingManagement"
-PORT = 3000
-JWT_SECRET = "your_jwt"
-CLOUDINARY_CLOUD_NAME = "dqrnqbx42"
-CLOUDINARY_API_KEY = "797465828361423"
-CLOUDINARY_API_SECRET = "81QBW9B75GkMx0r1NxwMeKIDo7k"
\ No newline at end of file
+MONGODB_URI = 
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)